integration/kubernetes/runtimeclass_workloads/pod-bad-health.yaml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: badhctest
+spec:
+  runtimeClassName: kata
+  containers:
+  - name: test
+    image: mhausenblas/simpleservice:0.5.0
```

Comment:
I wonder if this hard-coded image version is guaranteed to be retained or whether we need to drop the version number?

Reply:
@jodh-intel , unfortunately we can not drop the version number and put something like `mhausenblas/simpleservice` or `mhausenblas/simpleservice:latest` because we got the error of
`ErrImagePull` meaning that does not exist